src/app/settings/settings-routing.module.ts

```diff
@@ -3,20 +3,22 @@ import { RouterModule, Routes } from '@angular/router'
 import { SettingsComponent } from './components/settings/settings.component'
 import { ClearCacheComponent } from './components/clear-cache/clear-cache.component'
 
-export enum SettingsPaths {
+export enum SettingsPath {
   SETTINGS = '',
-  KEYBOARD_MAPPINGS = 'keyboard-mappings',
-  CLEAR_CACHE = 'clear-cache',
+  KEYBOARD_MAPPINGS = 'KEYBOARD_MAPPING',
+  SHELF = 'SHELF',
+  CLEAR_CACHE = 'CLEAR_CACHE',
```

Comment:
Did we intentionally make the paths all uppercase? To me, this URL address seems rather strange.

Reply:
I changed them to uppercase before I noticed they were shown in the URL.
I agree with you, it looks weird in the URL. 
I have changed the value back to be lowercased.